core/modules/layout/layout.admin.inc

```diff
@@ -74,26 +130,71 @@ function layout_list_page() {
       if ($layout->disabled) {
         $class[] = 'disabled';
       }
-      $rows[] = array('data' => $row, 'class' => $class);
+
+      if ($path !== '' && $page_callbacks[$path] == NULL) {
+        // Add the disabled class to broken rows.
+        $class[] = 'disabled';
+        $broken_rows[] = array('data' => $row, 'class' => $class);
+      }
+      elseif (isset($page_callbacks[$path]) && $page_callbacks[$path] == 'layout_page_callback') {
+        $stand_alone_rows[] = array('data' => $row, 'class' => $class);
+      }
+      else {
+        $override_rows[] = array('data' => $row, 'class' => $class);
+      }
     }
   }
 
   $header = array(
-    array('data' => t('Path and layout'), 'class' => array('layout-title')),
+    array('data' => t('Layout'), 'class' => array('layout-title')),
     array('data' => t('Visibility conditions'), 'class' => array('layout-conditions', 'priority-low')),
     array('data' => t('Storage state'), 'class' => array('layout-status', 'priority-low')),
     array('data' => t('Operations'), 'class' => array('layout-operations')),
   );
 
   $page = array();
   $page['#attached']['css'][] = backdrop_get_path('module', 'layout') . '/css/layout.admin.css';
-  $page['table'] = array(
+  $page['stand_alone'] = array(
+    '#type' => 'help',
+    '#markup' => '<h2>' . t('Layout pages') . '</h2>' . t('These layouts will create new stand-alone pages on the site. Without the layouts, these pages would not exist.'),
+    '#weight' => 0,
+  );
+  $page['stand_alone_table'] = array(
     '#theme' => 'table',
     '#header' => $header,
-    '#rows' => $rows,
+    '#rows' => $stand_alone_rows,
     '#attributes' => array('class' =>   array('layout-list')),
     '#empty' => t('No layouts have been created yet.'),
+    '#weight' => 1,
   );
+  $page['override'] = array(
+    '#type' => 'help',
+    '#markup' => '<h2>' . t('Layout overrides') . '</h2>' . t('These layouts will override existing pages on the site. Without the layouts, these pages would use other layouts.'),
+    '#weight' => 2,
+  );
+  $page['override_table'] = array(
+    '#theme' => 'table',
+    '#header' => $header,
+    '#rows' => $override_rows,
+    '#attributes' => array('class' =>   array('layout-list')),
+    '#weight' => 3,
+  );
+
+  if (!empty($broken_rows)) {
+    $page['broken'] = array(
+      '#type' => 'help',
+      '#markup' => '<h2>' . t('Broken Layouts') . '</h2>' . t('These layouts cannot be used unless the module that provides the path is enabled.'),
```

Comment:
Should be `t('Broken layouts')` for capitalization consistency with other two headings.

Reply:
Fixed, and rebased PR off latest 1.x